base/database/__init__.py

```diff
@@ -32,78 +32,123 @@ def download_fname(download_path: str, download_url: str):
                       download_url.split("/")[-1])
 
 
-def initialize_sqlite_database(sel_wormbase_version,
+def initialize_postgres_database(sel_wormbase_version,
                                strain_only=False):
-  """Create a static sqlite database
+  """Create a postgres database
   Args:
         sel_wormbase_version - e.g. WS276
 
-  Generate an sqlite database
+  Generate a postgres database
   """
-  start = arrow.utcnow()
   console.log("Initializing Database")
-
   DATASET_RELEASE = config['DATASET_RELEASE']
-  SQLITE_PATH = f"base/cendr.{DATASET_RELEASE}.{sel_wormbase_version}.db"
-  SQLITE_BASENAME = os.path.basename(SQLITE_PATH)
 
   # Download wormbase files
   if strain_only is False:
-    if os.path.exists(SQLITE_PATH):
-      os.remove(SQLITE_PATH)
-
-    if not os.path.exists(DOWNLOAD_PATH):
-      os.makedirs(DOWNLOAD_PATH)
-
-    # Parallel URL download
-    console.log("Downloading Wormbase Data")
-    GENE_GFF_URL = URLS.GENE_GFF_URL.format(WB=sel_wormbase_version)
-    GENE_GTF_URL = URLS.GENE_GTF_URL.format(WB=sel_wormbase_version)
-    download([URLS.STRAIN_VARIANT_ANNOTATION_URL,
-              GENE_GFF_URL,
-              GENE_GTF_URL,
-              URLS.GENE_IDS_URL,
-              URLS.HOMOLOGENE_URL,
-              URLS.ORTHOLOG_URL,
-              URLS.TAXON_ID_URL],
-              DOWNLOAD_PATH)
-
-    sva_fname = download_fname(DOWNLOAD_PATH,URLS.STRAIN_VARIANT_ANNOTATION_URL)
-    gff_fname = download_fname(DOWNLOAD_PATH, GENE_GFF_URL)
-    gtf_fname = download_fname(DOWNLOAD_PATH, GENE_GTF_URL)
-    gene_ids_fname = download_fname(DOWNLOAD_PATH, URLS.GENE_IDS_URL)
-    homologene_fname = download_fname(DOWNLOAD_PATH, URLS.HOMOLOGENE_URL)
-    ortholog_fname = download_fname(DOWNLOAD_PATH, URLS.ORTHOLOG_URL)
+    f = download_external_data(sel_wormbase_version)
 
   from base.application import create_app
   app = create_app()
   app.app_context().push()
-  app.config['SQLALCHEMY_DATABASE_URI'] = f"sqlite:///{SQLITE_BASENAME}"
+  app.config['SQLALCHEMY_DATABASE_URI'] = f'postgresql://admin:password@localhost/cendr'
```

Comment:
Avoid hardcoding credentials in the repo. Load them from ENV variables/Google Secrets/IAM. (https://cloud.google.com/sql/docs/postgres/iam-logins)

Consider loading the DB credentials dynamically for each environment (prod, local).

Reply:
This DB is generated on a local Postgres instance and then dumped to a .sql file, which gets uploaded to google bucket and then imported into the cloudsql instance - this username/pass is not a secret, they are the creds to the temporary local db i spin up to build the tables instead of executing every query against the cloud sql DB (also if I used admin/password in production that would be a WHOLE other issue :D)